Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WeatherLiveTileAndToast/WeatherLiveTileAndToastCode.bind

```diff
@@ -121,7 +121,7 @@ private static bool IsAdaptiveToastSupported()
 {
     switch (AnalyticsInfo.VersionInfo.DeviceFamily)
     {
-        // Desktop and Mobile started supporting adaptive toasts in API contract 3 (Anniversary Update)
+        // Desktop and Mobile started supporting adaptive toasts in API contract 3 (Anniversary Update)	
         case "Windows.Mobile":
         case "Windows.Desktop":
             return ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 3);
```

Comment:
Do we need to check for this ApiContract?

Reply:
This is the bind file, and the notifications package goes all the way back to the first version of windows 10, so yes, we do.